### DISCUSSION TITLE
Accept both comma and dot as decimal separator

### DISCUSSION THREAD

**Discussion by vmoriaux** [hargata/lubelog#611] [Ideas]
I'm running LubeLogger using the FR local. This means the decimal separator is "," (comma). Yet when entering the cost or any numerical value, I use my numpad which only contains a "." for the decimal separator.
Most office applications (calc, spreadsheet, etc.) accept both separators for input fields. So I use the "." and LubeLogger silently discard the value (defaults to 0 for the cost for example). So I need to be careful and watch my fingers to make sure I use the comma and not the dot. Then double check that the cost (for example) is persisted after I create a new service record (for example) just in case I automatically/mechanically used the dot as I do in any other app.

**Comment by hargata** [comment]
In the next release, we added a feature called "Automatic Decimal Formatting" which will allow you to enter costs without ever touching the decimal key.

See Examples:

US/UK Decimal Format with "." as separator:

![automaticnumberformat_US](https://github.com/user-attachments/assets/263d6cbc-32dd-4286-9a6a-0df1c448ee5e)

EU Format with "," as separator:
![automaticnumberformat_EU](https://github.com/user-attachments/assets/1c19c49f-2714-4df0-9771-c7c82dcc8a55)

**Reply by vmoriaux** [reply]
great, I can't wait to test it. Thanks

**Reply by hargata** [reply]
This is released, can be tested on the demo site.
